src/searchengine/searchengine.cpp

```diff
@@ -317,7 +317,24 @@ void SearchEngine::downloadFinished(int exitcode, QProcess::ExitStatus) {
     delete downloadProcess;
 }
 
-static void removePythonScriptIfExists(const QString& script_path)
+static inline void removeDirRecursive(const QString& dirName) {
+    QDir dir(dirName);
+
+    if (!dir.exists()) return;
+
+    Q_FOREACH(QFileInfo info, dir.entryInfoList(QDir::NoDotAndDotDot |
+                                                QDir::System |
+                                                QDir::Hidden |
+                                                QDir::AllDirs |
+                                                QDir::Files, QDir::DirsFirst)) {
+        if (info.isDir()) removeDirRecursive(info.absoluteFilePath());
+        else QFile::remove(info.absoluteFilePath());
```

Comment:
fsutils::forceRemove()
Also move the whole function to fsutils. It could be useful elsewhere in the future.